RFUSB_to_DB15/drivers.cpp

```diff
@@ -46,7 +47,7 @@ void setupController(uint16_t vid, uint16_t pid, HIDController *controller) {
       break;
 
     case VID_BROOK:
-      if (pid == PID_BROOK_UNIVERSAL) setupPS4(controller);
+      if (pid == PID_BROOK_UNIVERSAL || PID_BROOK_FB) setupPS4(controller);
```

Comment:
This statement is always true. 
please change this to:
`if ((pid == PID_BROOK_UNIVERSAL) || (pid == PID_BROOK_FB)) setupPS4(controller);`

Reply:
And you know what is odd, it's working with both Brook PCBs.

Reply:
Yeah it works because they both use the same driver.  The original code just said:
"If the VID is brook, set up a PS4 controller"  and since they both worked like a PS4 controller it was fine, but if another controller by Brook came along that wasn't a PS4 controller it would try to set it up as a PS4 controller and weird things would happen.